import exception.TaskException;
import exception.TestException;

/**
 * Created by sloanzhang on 2019/4/18.
 */
public class TaskDao {

    public static final int ERROR_PRAR=1;
    public static final int ERROR_SERVEICE=2;
    public int createTask() throws TaskException {
        return 0;
    }


    public int startTask() throws TestException {
        return 0;


    }
}
